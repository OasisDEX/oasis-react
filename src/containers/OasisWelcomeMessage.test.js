/* global shallow describe it expect */
/* eslint-disable import/first */
import React from 'react';

import {
  OasisWelcomeMessageWrapper,
  mapStateToProps,
  mapDispatchToProps
} from './OasisWelcomeMessage';
import { shallow } from 'enzyme';

describe('(Container) OasisWelcomeMessage', () => {
  const initialActions = mapDispatchToProps(x => x);
  const props = {
    ...initialActions,
  };

  it('will receive right actions', () => {
    expect(initialActions).toMatchSnapshot();
  });

  it('should render', () => {
    const wrapper = shallow(
      <OasisWelcomeMessageWrapper {...props}/>
    );
    expect(wrapper).toMatchSnapshot();
  });

});
