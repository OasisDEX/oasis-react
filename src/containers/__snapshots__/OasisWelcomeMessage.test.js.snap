// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Container) OasisWelcomeMessage should render 1`] = `
<OasisMessage
  dismissible={true}
  heading="Get started!"
  onDismiss={[Function]}
  type="MSGTYPE_INFO"
>
  <ul>
    <li>
      <span
        styleName="Circle"
      >
        <span
          styleName="Number"
        >
          1
        </span>
      </span>
      <span
        styleName="Text"
      >
        Go to wrap/unwrap and wrap ETH to turn it into W-ETH.
      </span>
    </li>
    <li>
      <span
        styleName="Circle"
      >
        <span
          styleName="Number"
        >
          2
        </span>
      </span>
      <span
        styleName="Text"
      >
        Go to Trade and place new orders or trade existing orders by clicking on an open order on the order book to buy/sell into it.
      </span>
    </li>
    <li>
      <span
        styleName="Circle"
      >
        <span
          styleName="Number"
        >
          3
        </span>
      </span>
      <span
        styleName="Text"
      >
        The first time you make a transaction you will be prompted to set your allowance.
      </span>
    </li>
  </ul>
</OasisMessage>
`;

exports[`(Container) OasisWelcomeMessage will receive right actions 1`] = `
Object {
  "actions": Object {
    "dismissMessage": [Function],
  },
}
`;
